src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_LENGTH = 10;
+    private final Random random = new Random();
+    private final ColorSupplier randomColor = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(Figures.values().length);
+        Figures figureType = Figures.values()[index];
+
+        if (figureType == Figures.SQUARE) {
```

Comment:
If you have defined values, you can use switch case in this place

Reply:
Done